js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired ('fetchOrders requires the symbol parameter');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'pair': market['id'],
+            // 'startTimestamp': 0,
+            // 'endTimestamp': 0,
+            // 'statusKind': '',
+            // 'orderId': '',
+        };
+        if (since !== undefined) {
+            request['startTimestamp'] = since;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privateGetOrdersAllPair (this.extend (request, params), params);
+        const orders = this.safeValue (response, 'data', []);
+        //
+        //     {
+        //         "data":[
+        //             {
+        //                 "id":"2220595581",
+        //                 "pair":"bnb_twd",
+        //                 "price":"15000",
+        //                 "avgExecutionPrice":"0",
+        //                 "action":"SELL",
+        //                 "type":"LIMIT",
+        //                 "createdTimestamp":1644896744886,
+        //                 "updatedTimestamp":1644898706236,
+        //                 "status":4,
+        //                 "originalAmount":"0.01",
+        //                 "remainingAmount":"0.01",
+        //                 "executedAmount":"0",
+        //                 "fee":"0",
+        //                 "feeSymbol":"twd",
+        //                 "bitoFee":"0",
+        //                 "total":"0",
+        //                 "seq":"BNBTWD8540871774",
+        //                 "timeInForce":"GTC"
+        //             }
+        //         ]
+        //     }
+        //
+        return this.parseOrders (orders, market, since, limit);
+    }
+
+    fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        const request = {
+            'statusKind': 'OPEN',
+        };
+        return this.fetchOrders (symbol, since, limit, this.extend (request, params));
+    }
+
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        const request = {
+            'statusKind': 'DONE',
+        };
+        return this.fetchOrders (symbol, since, limit, this.extend (request, params));
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired ('fetchMyTrades requires the symbol parameter');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'pair': market['id'],
+        };
+        const response = await this.privateGetOrdersHistory (this.extend (request, params));
```

Comment:
Can you point me to the docs of this endpoint? Can't find them :/ 
Second, are you sure this retrieves trades? From what I could test seems to return past orders only

Reply:
I didn't see this too. The api info came from one of their employee (my firend).

Reply:
I found the doc: https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v2/rest/auth/history.md

It's the old api, and return user's order.